diff --git a/volume_manager/PublicVolume.cpp b/volume_manager/PublicVolume.cpp
index b04135fd07a99669ffd5967281eb509af4af8175..cea34ffcf30a6993e305ab98632e61c826401609 100644
--- a/volume_manager/PublicVolume.cpp
+++ b/volume_manager/PublicVolume.cpp
@@ -31,6 +31,7 @@
 #include <cutils/fs.h>
 #include <private/android_filesystem_config.h>

+#include <algorithm>
 #include <fcntl.h>
 #include <stdlib.h>
 #include <sys/mount.h>
diff --git a/volume_manager/include/volume_manager/VolumeManager.h b/volume_manager/include/volume_manager/VolumeManager.h
index c2cb8dde9b0c6adbd51bd8f69e7d76cd2caf1a3d..c8cf97000bb4a760b2b3dc7b98102f477852531c 100644
--- a/volume_manager/include/volume_manager/VolumeManager.h
+++ b/volume_manager/include/volume_manager/VolumeManager.h
@@ -23,6 +23,7 @@
 #include <stdlib.h>

 #include <list>
+#include <vector>
 #include <mutex>
 #include <string>
 
