diff --git a/Android.bp b/Android.bp
index 285475c0ee4292f35a63493a5a0a716d8a70eec4..2770b554b629464e68dd7f5ba1b95cc9659c8d90 100644
--- a/Android.bp
+++ b/Android.bp
@@ -16,6 +16,7 @@ license {
 cc_library {
     name: "zuid_md5",
     host_supported: true,
+    vendor_available: true,
     sdk_version: "current",
     stl: "c++_shared",
     target: {
@@ -37,6 +38,7 @@ cc_library {
 cc_library {
     name: "xmp_toolkit_sdk",
     host_supported: true,
+    vendor_available: true,
     sdk_version: "current",
     stl: "c++_shared",
     export_include_dirs: [
