diff --git a/services/config/src/device_impl.h b/services/config/src/device_impl.h
index 4c22a47d86bc4514fcbf9e14b5984859dfcba8fe..e2f1f70fb53bb356cbdced36e9771e6c157da479 100644
--- a/services/config/src/device_impl.h
+++ b/services/config/src/device_impl.h
@@ -35,6 +35,7 @@
 #include <log/log.h>
 #include <config/device_interface.h>
 #include <map>
+#include <mutex>
 #include <utility>
 #include <string>
 #include <vector>
